backend/src/jobs_server/models.py

```diff
@@ -111,3 +123,9 @@ class LogOptions(BaseModel):
         default=-1,
         description="Number of tail lines of logs, -1 for all",
     )
+
+
+class ListWorkloadModel(BaseModel):
```

Comment:
Could you explain why we define another model? From the code I see that we construct models (status, ID) from Kueue workloads directly. Could that just be done on the client side, or is there a tangible benefit of doing things this way instead of just returning a `list[KueueWorkload]`?

Reply:
As discussed, it was mostly a "cost-saving" measure, if we only want to return the summary information. If we want to move more processing to the client-side, we could in theory also return `list[KueueWorkload]`, but that means that we'll have to duplicate the logic that extracts the managed workload etc in the client as well. (I would like to keep that business logic constrained to the backend, if possible).

If you think we should return different information (but I think from our call yesterday we agreed that this is sufficient), we can always change this model.